src/styles/components/expanding-table/styles.less

```diff
@@ -85,6 +85,10 @@
 
   .expanding-table-child {
 
+    > div {
```

Comment:
the js solution in place isn't working I have tried to use another ways through js but none of them worked so I moved to the simple fix since this is a very particular html structure (one depending on another) specifying the child div is sufficient.